src/test/resources/features/Sessions.feature

```diff
@@ -0,0 +1,315 @@
+    Then Im on "Manage prison visits - Vsip_prisoner13, Do Not Use" page
+    # This Prisoner is based in Reception so Booking  cannot be booked
+    And I check Prisoner location "RECP, Bristol (HMP)" unable to proceed to book a visit
+    And I click on Book a visit button
+    Then Im on "Manage prison visits - Select visitors from the prisoner’s approved visitor list" page
+    And I select a visitor the list from Bristol establishment
+    And click on continue button
+    Then Im on "Manage prison visits - Select date and time of visit" page
+    #Prisoner Location in Reception so time slots available
+    And I see I cannot go further no time slots is available
+    And I sign out of the service
+
+    Examples:
+      | userName  | password          | prisonNumber |
+      | VSIP4_TST | DigitalServices10 | A8898DZ      |
+
+  @suite
+  Scenario Outline: Incentive level - User check incentive level within an establishment where the Prisoner is based
+  # ensure a session restricted by incentive level is only available to prisoners with that incentive level
+
+    Given I navigate to VSIP service
+    And Im on "HMPPS Digital Services - Sign in" page
+    And I enter "<userName>"
+    And I enter the "<password>"
+    And click on submit button
+    Then Im on "Manage prison visits - Manage prison visits" page
+    And I select change establishment
+    And I change the establishmnet to Bristol
+    And click on continue button
+    And I click on Book a visit option
+    And Im on "Manage prison visits - Search for a prisoner" page
+    When I enter "<prisonNumber>" to search for a prison
+    And click on search button
+    Then I choose prison from search results
+    Then Im on "Manage prison visits - Vsip_prisoner13, Do Not Use" page
+    And I check incentive level for that prisoner
+    And I sign out of the service
+
+    Examples:
+      | userName  | password          | prisonNumber |
+      | VSIP4_TST | DigitalServices10 | A8898DZ      |
+
+  @suite
+  Scenario Outline: Incentive level - User check on incentive level of a Prisoner within an establishment
+  # ensure a session restricted by incentive level is not available to a prisoner with that incentive level
+
+    Given I navigate to VSIP service
+    And Im on "HMPPS Digital Services - Sign in" page
+    And I enter "<userName>"
+    And I enter the "<password>"
+    And click on submit button
+    Then Im on "Manage prison visits - Manage prison visits" page
+    And I select change establishment
+    And I change the establishmnet to Bristol
+    And click on continue button
+    And I click on Book a visit option
+    And Im on "Manage prison visits - Search for a prisoner" page
+    When I enter "<prisonNumber>" to search for a prison
+    And click on search button
+    Then I choose prison from search results
+    Then Im on "Manage prison visits - Vsip_prisoner12, Do Not Use" page
+    And I check enhanced incentive level for that prisoner
```

Comment:
tests missing for checking for slots by incentive level. we only seem to be chcking if prisoner has enhanced incentive level, nothing further.